packages/commons/src/main/util/Runtime.ts

```diff
@@ -43,17 +43,28 @@ export class Runtime {
   }
 
   public static getOSFamily(): OperatingSystem {
+    const OS = {
+      DESKTOP: {
+        LINUX: ['linux', 'ubuntu', 'fedora', 'gentoo', 'debian', 'suse', 'centos', 'red hat', 'freebsd', 'openbsd'],
+        MAC: ['os x', 'mac os'],
+        WINDOWS: ['windows', 'windows server 2008 r2 / 7', 'windows server 2008 / vista', 'windows xp'],
```

Comment:
Isn't it a bit redundant to include all those names, as all of those contain the word `windows`?